partiql-ast/build.gradle.kts

```diff
@@ -16,6 +16,7 @@
 plugins {
     id(Plugins.conventions)
     id(Plugins.publish)
+    id(Plugins.kotlinLombok) version Versions.kotlinLombok
```

Comment:
What's the purpose of this plugin? From what I can tell online, it's not required for using the annotations.

Reply:
I was following the Kotlin Lombok sample code, which had included the plugin -- https://github.com/kotlin-hands-on/kotlin-lombok-examples/blob/master/kotlin_lombok_gradle/nokapt/build.gradle.kts.

From what I can tell, it's just used to enable using Java Lombok code from Kotlin -- https://kotlinlang.org/docs/lombok.html:

> The Kotlin Lombok compiler plugin allows the generation and use of Java's Lombok declarations by Kotlin code in the same mixed Java/Kotlin module. If you call such declarations from another module, then you don't need to use this plugin for the compilation of that module.

Since I'm not sure we have such usages in the Kotlin AST code, I'll just remove for now. We can add it back if we ever need it.